Src/pch.h

```diff
@@ -83,6 +83,8 @@
 #pragma warning(pop)
 #endif
 
+#define _XM_NO_XMVECTOR_OVERLOADS_
```

Comment:
This just makes sure that when you are building with PC/Xbox, new code doesn't get written that would fail to build on clang due to the DirectXMath issues.

This define is used by DirectXMath 3.12 or later which is in the Windows 10 SDK (16299) or later and Xbox One XDK 2017 releases.

It's ignored by earlier versions of DirectXMath.